pydantic_core/core_schema.py

```diff
@@ -14,6 +14,8 @@
 else:
     from typing import Literal, TypedDict
 
+from pydantic_core._pydantic_core import ValidatorInfo
```

Comment:
Could be the problem?

Maybe relative import, remove, move here, or typing only import.

Reply:
It was